packages/dd-trace/src/debugger/devtools_client/breakpoints.js

```diff
@@ -48,6 +48,10 @@ async function addBreakpoint (probe) {
   const { url, scriptId, sourceMapURL, source } = script
 
   if (sourceMapURL) {
+    log.debug(
+      '[debugger:devtools_client] Translating location using source map for %s:%d:%d (probe: %s, version: %d)',
+      file, lineNumber, columnNumber, probe.id, probe.version
+    );
```

Comment:
oopsie ?
```suggestion
    )
```

Reply:
or wait, is it just required because next line is a parenthesis ? so the ASI fails ?

Reply:
Correct. I was contemplating moving to the beginning of the next line to make it more clear though. Do you want me to do that?